src/SUMMARY.md

```diff
@@ -67,6 +67,22 @@
 
 - [Type system](type-system.md)
     - [Types](types.md)
+        - [Boolean type](types/boolean-type.md)
+        - [Numeric types](types/numeric-type.md)
+        - [Textual types](types/textual-type.md)
+        - [Never type](types/never-type.md)
```

Comment:
What do you think about just adding a section to the never page that explains that it is only valid in function return type positions?

Otherwise, I would think #206 should be reverted, and the grammar updated so that function return types are `_Type_ | !`.

Also, it looks like there was some discussion on that PR about retaining the information about divergence (and #271), but that never happened?